cas-server-webapp-support/src/main/java/org/jasig/cas/web/flow/InitialFlowSetupAction.java

```diff
@@ -101,6 +107,11 @@ protected Event doExecute(final RequestContext context) throws Exception {
                         registeredService.getId());
                 WebUtils.putRegisteredService(context, registeredService);
             }
+        } else if (!this.enableFlowOnAbsentServiceRequest) {
```

Comment:
I'mp surprised by this change: I don't see the old behaviour I had in mind, i.e. it would not work without a `service` parameter.